packages/manager/core/api/src/client.ts

```diff
@@ -42,25 +43,44 @@ export const v6 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/apiv6',
 });
-v6.interceptors.response.use(null, handleAuthenticationError);
 
 export const aapi = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/2api',
 });
-aapi.interceptors.response.use(null, handleAuthenticationError);
 
 export const ws = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/ws',
 });
-ws.interceptors.response.use(null, handleAuthenticationError);
 
 export const v2 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/api/v2',
 });
-v2.interceptors.response.use(null, handleAuthenticationError);
 
-export const apiClient = { v6, aapi, ws, v2 };
+export const apiClient: any = { v6, aapi, ws, v2 };
+
+Object.keys(apiClient).forEach((api) => {
+  // Configure all headers with request tagger
+  apiClient[api]?.interceptors?.request.use(function(config: any) {
+    // add currentPage header to api call to fill X-OVH-MANAGER-PAGE
+    if (config?.headers?.currentPage) {
+      defineCurrentPage(config.headers.currentPage);
+    }
+
+    console.info('config?.baseUrl : ', config?.baseURL);
+    const headers = getHeaders(config?.baseURL);
+    console.info('api client headers : ', headers);
+    Object.keys(headers).forEach((header) => {
```

Comment:
can you return using an object spread instead of iterating ?
```
return {
  ...config,
  headers: {
    ...config.headers,
    ...headers,
  }
}
```